vector/src/main/java/im/vector/app/features/settings/VectorSettingsActivity.kt

```diff
@@ -77,13 +79,19 @@ class VectorSettingsActivity : VectorBaseActivity<ActivityVectorSettingsBinding>
                     replaceFragment(views.vectorSettingsPage, VectorSettingsAdvancedSettingsFragment::class.java, null, FRAGMENT_TAG)
                 SettingsActivityPayload.SecurityPrivacy ->
                     replaceFragment(views.vectorSettingsPage, VectorSettingsSecurityPrivacyFragment::class.java, null, FRAGMENT_TAG)
-                SettingsActivityPayload.SecurityPrivacyManageSessions ->
+                SettingsActivityPayload.SecurityPrivacyManageSessions -> {
+                    val fragmentClass = if (vectorPreferences.isNewSessionManagerEnabled()) {
+                        im.vector.app.features.settings.devices.v2.VectorSettingsDevicesFragment::class.java
```

Comment:
Maybe add the import to avoid this long line?

Reply:
I guess we can't avoid that. We have two `VectorSettingsDevicesFragment`, the V2 version and the previous one. This is only temporary. Once the new session manager is delabsed, we will remove all the old code.

Reply:
Ah, OK, I did not see that the names were identical.
FTR you could have used [import alias](https://kotlinlang.org/docs/packages.html#imports)
So:
```kotlin
import im.vector.app.features.settings.devices.v2.VectorSettingsDevicesFragment as VectorSettingsDevicesFragmentV2
```
but not now, this is fine, please do not update the PR!